.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+# About email notifications, see <https://git.io/JUo27>.
+
+on:
+  push:
+    # Run in PRs and for bors, but not on master.
+    branches:
+      - 'auto'
+      - 'try'
+      - '!gh-pages'
+  pull_request:
+    branches:
+      - 'master'
+  # TODO: fill the cron here
+  # schedule:
+  #   - cron: "30 0 * * 1" # Every Monday at half past midnight UTC skip-pr skip-stable
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
```

Comment:
Do you know a good introduction to GH action syntax? For example, where is this `matrix.os` variable defined?

Reply:
It is defined by the `matrix` starting from line 26 to 37.